app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
+  def create
+    @user = current_user
+    @post = Post.find(params[:post_id])
+    @comment = @user.comments.new(comment_params)
+    @comment.user_id = @user.id
+    @comment.post_id = params[:post_id]
+    @comment.save
+    redirect_to user_post_path(@user.id, @post)
+  end
+
```

Comment:
- [ ] Please kindly include a `new` controller method to the `CommentsController`.